.gitlab-ci.yml

```diff
@@ -60,11 +60,11 @@ init:
 
 .upstream_rules_all:
   rules:
-    - if: '$CI_PIPELINE_SOURCE != "schedule"'
+    - if: '$CI_PIPELINE_SOURCE != "schedule" && $RUNNER !~ /[\S]+rhel-[\S]+-(?:(?:ga)|(?:eus))[\S]+/'
 
 .upstream_rules_x86_64:
   rules:
-    - if: '$CI_PIPELINE_SOURCE != "schedule" && $RUNNER =~ "/^.*(x86_64).*$/"'
+    - if: '$CI_PIPELINE_SOURCE != "schedule" && $RUNNER =~ "/^.*(x86_64).*$/" && $RUNNER !~ /[\S]+rhel-[\S]+-(?:(?:ga)|(?:eus))[\S]+/'
```

Comment:
> It is differentiated by having NIGHTLY_GA variable instead of NIGHTLY.

I see only 2 states: either testing latest nightly compose or a GA compose and there is already an environment variable which can represent 2 states (3 actualy). Unless we really need another variable let's not add one and reuse whatever already exists. `NIGHTLY == "false"` is the same as `GA == "true"`, isn't it ?

Reply:
>  NIGHTLY_GA

On naming: this is mighty confusing. The names are chosen based on which is the subkect under test, not when the pipelines execute. As a matter of fact they seem to execute around 4-6 AM in the morning which you can argue in the summer is during the day.